<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>markingDash</class>
 <widget class="QDialog" name="markingDash">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Activity ID:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_goSubmissionPath">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Student's Submission</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Activity Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_actName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_students">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>111</width>
     <height>581</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Student ID</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>130</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Path of the Commended Student's Submission</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="file_Directory_SubmissionPath">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>70</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Solution">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Open The Solution</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SaveFile">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>670</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save The File</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Path of the Student's Submission</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_activityID">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="file_Directory_commendedFilePath">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>120</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Marking as: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_status">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>660</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>status</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_receiveData">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="Rubric_table">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>531</width>
     <height>331</height>
    </rect>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <column>
    <property name="text">
     <string>Item Number</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Item ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Item Text</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Value</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_studentID">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>511</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>691</x>
     <y>585</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Grade</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ItemNumber">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>570</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_studentGrade">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>170</y>
     <width>311</width>
     <height>331</height>
    </rect>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>ItemNumber</string>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ItemID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Grade</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Override</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>511</x>
     <y>511</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>StudentID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_GradeStatus">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>510</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>511</x>
     <y>540</y>
     <width>69</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Rubric Item ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>511</x>
     <y>570</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Item Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ItemID">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>540</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_updateGrade">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>590</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Grade">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>590</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
